src/lib/functions/server.mjs

```diff
@@ -222,9 +222,9 @@ export const startFunctionsServer = async (options) => {
       functionsDirectories.push(distPath)
     }
   } else {
-    // The order of the function directories matters. Leftmost directories take
+    // The order of the function directories matters. Rightmost directories take
```

Comment:
Hmm, this feels unrelated to the extension precedence. Is this a separate problem you've identified?

Reply:
But we're reversing the array of the functions that come out, not the list of directories going in?

(Sorry if I'm missing the point.)